frontend/src/components/atoms/inputs/Input.js

```diff
@@ -20,30 +20,38 @@ export const Input = ({
   // we generate a unique id prefixed by the field name
   const [id] = useState(uniqueId(name));
 
-  return (
-    <div className="fr-input-group">
-      <Label
-        id={id}
-        label={label}
-        required={required}
-        helper={helper}
-        meta={meta}
-      />
-      <input
-        className="fr-input"
-        type={type}
-        onChange={onChange}
-        name={name}
-        placeholder={placeholder}
-        id={id}
-        readOnly={disabled}
-        value={value}
-        aria-label={ariaLabel}
-        required={required}
-        {...rest}
-      />
-    </div>
+  const getInput = () => (
+    <input
+      className="fr-input"
+      type={type}
+      onChange={onChange}
+      name={name}
+      placeholder={placeholder}
+      id={id}
+      readOnly={disabled}
+      value={value}
+      aria-label={ariaLabel}
+      required={required}
+      {...rest}
+    />
   );
+
+  if (label) {
```

Comment:
The usage of input without label is forbidden by the dsfr and make the input not accessible. This usage should not be encouraged in my opinion. I'd rather use the <input> react element directly in the table header 🤔